### PR TITLE
STCOR-603 show supplemental data on the /settings/about page

### PR DESCRIPTION
**Pull request by zburke**
Optionally decorate the `/settings/about` (Settings: Software versions)
page with information from Okapi Environment variables,
mod-configuration values, or stripes.config, in descending order. Three
values may be displayed: a version, an install date, and a message.

The Okapi Environment keys are `ABOUT_INSTALL_DATE`,
`ABOUT_INSTALL_VERSION`, and `ABOUT_INSTALL_MESSAGE`. The mod-configuration
values all correspond to `module`: `@folio/stripes-core`, `configName`:
`aboutInstall` and the `code`s `date`, `version`, and `message`. The
`stripes.config.js` values are from the `stripes.config` object with the
keys `aboutInstallDate`, `aboutInstallVersion`, and
`aboutInstallMessage`.

Refs [STCOR-603](https://issues.folio.org/browse/STCOR-603), NH-603 (No taxes, no services! Live Free or Die!)

### CODE REVIEW

File: src/queries/useConfigurations.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+};
+
+const useConfigurations = ({ module, configName, code }) => {
+  const stripes = useStripes();
+  const ky = useOkapiKy();
+  const namespace = '@folio/stripes-core:QUERY_KEY_CONFIGURATIONS';
+
+  const { isFetching, data, error } = useQuery(
+    [namespace, configName, code],
+    async () => {
```
Comment (mkuklis) [inline src/queries/useConfigurations.js:32]: @zburke I don't think we need `async` here since there is no `await` right?

Comment (mkuklis) [reply src/queries/useConfigurations.js:32]: Or perhaps the `await` is missing before `ky.get`?

**Review by mkuklis — APPROVED** [review]
A clean, well tested and documented code ❤️ .

I left just one little comment.

File: src/queries/useConfigurations.js (hunk 1 above)
Comment (zburke) [reply src/queries/useConfigurations.js:32]: I think you're correct that `async` here is superfluous. It's `useQuery` that would be calling `await` here, but given `ky.get()` is already async there's no value in restating it. I'll remove it. 

Thanks for mentioning this and helping to keep the code as clean as possible!

**Pull request merged**